Library/DiscUtils.BootConfig/DiscUtils.BootConfig.csproj

```diff
@@ -6,19 +6,7 @@
     <AssemblyTitle>DiscUtils.BootConfig</AssemblyTitle>
     <Authors>Kenneth Bell;LordMike</Authors>
     <TargetFrameworks>netstandard1.5;net20;net40;net45</TargetFrameworks>
-    <AssemblyName>DiscUtils.BootConfig</AssemblyName>
-    <AssemblyOriginatorKeyFile>../../SigningKey.snk</AssemblyOriginatorKeyFile>
-    <SignAssembly>true</SignAssembly>
-    <PackageId>DiscUtils.BootConfig</PackageId>
     <PackageTags>DiscUtils;BootConfig</PackageTags>
-    <PackageProjectUrl>https://github.com/DiscUtils/DiscUtils</PackageProjectUrl>
-    <PackageLicenseUrl>https://github.com/DiscUtils/DiscUtils/blob/master/LICENSE.txt</PackageLicenseUrl>
-    <RepositoryType>git</RepositoryType>
-    <RepositoryUrl>https://github.com/DiscUtils/DiscUtils</RepositoryUrl>
-    <GenerateAssemblyConfigurationAttribute>false</GenerateAssemblyConfigurationAttribute>
-    <GenerateAssemblyCompanyAttribute>false</GenerateAssemblyCompanyAttribute>
-    <GenerateAssemblyProductAttribute>false</GenerateAssemblyProductAttribute>
-    <GeneratePackageOnBuild Condition="'$(Configuration)' != 'Debug'">True</GeneratePackageOnBuild>
```

Comment:
You seem to have removed way too many things from this csproj file ...

Reply:
I dropped the commit which contains these changes, they are not really relevant to this PR. But the values here are either default values, or values which are also defined in `common.props`.